pkg/frontend/adminactions/azureactions.go

```diff
@@ -102,3 +113,20 @@ func (a *azureActions) VMResize(ctx context.Context, vmName string, size string)
 	vm.HardwareProfile.VMSize = mgmtcompute.VirtualMachineSizeTypes(size)
 	return a.virtualMachines.CreateOrUpdateAndWait(ctx, clusterRGName, vmName, vm)
 }
+
+func (a *azureActions) AppLensGetDetector(ctx context.Context, detectorId string) ([]byte, error) {
+	resp, err := a.appLens.GetDetector(ctx, &applens.GetDetectorOptions{ResourceID: a.oc.Properties.ClusterProfile.ResourceGroupID, DetectorID: detectorId})
```

Comment:
`ResourceGroupID`?

Reply:
Updated to use cluster resource id, and not cluster resource group resource id

Reply:
Refer to [previous comment](https://github.com/Azure/ARO-RP/pull/2243#discussion_r920620112)